test/e2e-common.sh

```diff
@@ -375,16 +371,14 @@ function install() {
   fi
 
   if (( ENABLE_TLS )); then
-    echo "Patch to config-network to enable internal encryption"
-    toggle_feature system-internal-tls Enabled config-network
+    echo "Patch config-network to enable encryption features"
+    toggle_feature system-internal-tls enabled config-network
+
+    # This is currently only supported by kourier
     if [[ "$INGRESS_CLASS" == "kourier.ingress.networking.knative.dev" ]]; then
-      echo "Point Kourier local gateway to custom server certificates"
-      toggle_feature cluster-cert-secret server-certs config-kourier
-      # This needs to match the name of Secret in test/config/tls/cert-secret.yaml
-      export CA_CERT=ca-cert
-      # This needs to match $san from test/config/tls/generate.sh
-      export SERVER_NAME=knative.dev
+      toggle_feature cluster-local-domain-tls enabled config-network
```

Comment:
for kourier, all tests are now run with TLS also on cluster-local domains, this includes all our existing tests that call to `*.svc.cluster.local`